names/names.py

```diff
@@ -10,11 +10,11 @@
 names_2 = f.read().split("\n")  # List containing 10000 names
 f.close()
 
-duplicates = []
-for name_1 in names_1:
-    for name_2 in names_2:
-        if name_1 == name_2:
-            duplicates.append(name_1)
+#  use of set 
+#  makes is really fast
+#  https://www.programiz.com/python-programming/set 
+
```

Comment:
Super super great job here! I love this! This is how you get a O(n) time solution, with possibly a constant time space complexity!